src/controller/expenseController.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
+
+const expenseServise = require('../services/expenses.js');
+const userServise = require('../services/users.js');
+
+const post = (req, res) => {
+  const {
+    userId,
+    title,
+  } = req.body;
+
+  if (!title || !userServise.exist(userId)) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newExpense = {
+    ...req.body,
+    id: Math.floor(Math.random()),
+  };
+
+  expenseServise.add(newExpense);
+  res.statusCode = 201;
+  res.send(newExpense);
+};
+
+const get = (req, res) => {
+  const expenses = expenseServise.getAll();
+  const {
+    userId,
+    category,
+    from,
+    to,
+  } = req.query;
+
+  if (typeof +userId !== 'number') {
+    res.sendStatus(400);
+
+    return;
+  };
+
+  if (expenses.length === 0) {
+    res.send([]);
+  };
```

Comment:
Why did you do this check?( I asked, because just interested about purpose, not saying that it is wrong, just trying to understand)